### ISSUE TITLE
Feature request: Local PDF easier loading?

### ISSUE THREAD

**Issue by samharwood** [ken107/read-aloud#330]
It would be really helpful for us to be able to use ReadAloud to read exam papers if loading local PDFs files was easier.

Right now if I open a local PDF in the browser, then click ReadAloud, it shows the "Select PDF to Read Aloud" page and we have to pick the file again to load into ReadAloud. 
I know there are difficulties with CORS and local files. But is there any way around this? The "Allow access to file URLs" option in extensions doesn't seem to help at all (we are using Edge).

**Comment by samharwood** [comment]
Thanks. Your suggestion is a good workaround as I can provide this as a shortcut for staff to click on.

**Comment by samharwood** [comment]
Our exam accounts have to have internet disabled. I requested our IT to unblock https://assets.lsdsoftware.com/read-aloud/pdf-viewer-2/web/readaloud.html to allow PDF loading but they are unable to do so. 
Is there some way to include the PDF viewer in the extension so it can be used to read PDF's offline? Everything else _seems_ to work OK without internet access, except for being unable to load the PDF viewer. 
I've tried to see if I could build a custom version with viewer embeddded but I don't know how
